indexer-app/test/cmr/indexer/test/data/concepts/tag.clj

```diff
@@ -0,0 +1,30 @@
+(ns cmr.indexer.test.data.concepts.tag
+  "Code coverage tests for the functions of testing cmr.indexer.data.concepts.tag
+   namespace."
+  (:require
+   [clojure.string :as str]
+   [clojure.test :refer :all]
+   [cmr.common.util :as util]
+   [cmr.indexer.data.concepts.tag :as tag]))
+
+(defn- create-tag
+  "Create a tag map"
+  [tag-key originator data]
+  {:tag-key-lowercase (str/lower-case tag-key)
+   :originator-id-lowercase (util/safe-lowercase originator)
+   :tag-value-lowercase (when (string? data) (str/lower-case data))})
```

Comment:
Why only use `safe-lowercase` for one of the keys?

Reply:
it's how I saw the data was being processed in indexer-app/src/cmr/indexer/data/concepts/tag.clj, so I decided to mirror it here.